packages/core/src/external/aws/appConfig.ts

```diff
@@ -20,12 +20,20 @@ export type SandboxLimitFF = {
   cxIds: never;
 };
 
+export type EnabledFF = {
```

Comment:
nit: maybe rename it `BooleanFF` or `BasicFF`? Enabled conveys the meaning of something enabled.